.github/workflows/manual-publish-docs.yml

```diff
@@ -10,6 +10,12 @@ jobs:
       contents: write # Needed in this case to write github pages.
     steps:
       - uses: actions/checkout@v4
+
```

Comment:
Is the secret needed for the CI workflow? We may have problems when it comes to contributor PRs that cannot use secrets.